docs/scanner-quickstart.md

```diff
@@ -0,0 +1,187 @@
+
+## Scan Node Requirements
+
+The following are the requirements for running a Forta scan node.
+
+- 64-bit Linux distribution
+- CPU with 4+ cores
+- 16GB RAM
+- Connection to Internet
+- Docker v20.10+
+- **Recommended:** Ethereum Light Node
+
+## Install Forta
+
+The Forta scan node software is available for popular 64-bit Linux distributions using official Forta repositories. Package installation methods are verifiable (auto-verified during installation) and help you install required dependencies.
+
+### Install via YUM (CentOS, Fedora, Red Hat Enterprise Linux etc.)
+
+```
+$ sudo curl https://dist.forta.network/repositories/yum/Forta.repo -o /etc/yum.repos.d/Forta.repo -s
+$ sudo yum install forta
+```
+
+### Install via APT (Ubuntu, Debian etc.)
+
+```
+$ sudo curl https://dist.forta.network/pgp.public -o /usr/share/keyrings/forta-keyring.asc -s
+$ sudo echo 'deb [signed-by=/usr/share/keyrings/forta-keyring.asc] https://dist.forta.network/repositories/apt stable main' > /etc/apt/sources.list.d/forta.list
+$ sudo apt-get update
+$ sudo apt-get install forta
+```
+
+### Install Manually
+
+[Install Docker](https://docs.docker.com/get-docker/) (at least v20.10) 
+
+Download the latest x86-64 release binary and install
+
+```
+$ sudo curl https://dist.forta.network/artifacts/forta -o /usr/local/bin/forta
+```
+
+Make the binary executable
+
+```
+$ sudo chmod 755 /usr/local/bin/forta
+```
+
+
+## Initial Setup
+
+Forta scan node's CLI allows you to set up your first Forta configuration directory along with creating and managing your scan node's private key.
+
+### Initialize Forta Directory
+
+Initialization creates a private key that will sign the alerts from your scan node.  You must set the $FORTA_PASSPHRASE environment variable or provide the --passphrase flag to the `init` command.
+
+Initialize Forta using the `forta init` command
+
+```
+$ forta init --passphrase <your_passphrase>
+```
+
+This generates a config directory, a private key, and output your address
+
+```
+Scanner address: 0xAAA8C491232cB65a65FBf7F36b71220B3E695AAA
+
+Successfully initialized at /yourname/.forta
+```
+
+Take note of the address. This is the value that will be registered in the scan node registry smart contract.
+
+!!! note "Forta Directory"
+    By default, the forta directory is located in `~/.forta`. If you would like to use a different directory, either set the $FORTA_DIR env var or provide the `--dir` flag to the `init` command. Init command will initialize your Forta configuration and key to this directory.
+
+### Configure systemd
+
+If the binary ever stops, it must be restarted.  If you used a package installation method, there is a forta systemd service that can now be updated with your passphrase and config directory.
+
+The service definition can be found here: `/usr/lib/systemd/system/forta.service`
```

Comment:
It seems to reside in `/lib/systemd/system/forta.service` in Ubuntu - this wrong info is kind of my fault. After installation, though, `systemctl cat forta.service` should work to output this and the location (try with docker.service on your machine if interested :) )